vic/drivers/cesm/src/cesm_put_data.c

```diff
@@ -0,0 +1,295 @@
+/******************************************************************************
+ * @section DESCRIPTION
+ *
+ * Collect coupled fields and export to coupler
+ *
+ * Sign convention: positive value <=> downward flux
+ * Units: see notes in code or seq_flds_mod.F90
+ *
+ * @section LICENSE
+ *
+ * The Variable Infiltration Capacity (VIC) macroscale hydrological model
+ * Copyright (C) 2014 The Land Surface Hydrology Group, Department of Civil
+ * and Environmental Engineering, University of Washington.
+ *
+ * The VIC model is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; either version 2
+ * of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along with
+ * this program; if not, write to the Free Software Foundation, Inc.,
+ * 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *****************************************************************************/
+
+#include <vic_driver_cesm.h>
+
+void
+vic_cesm_put_data()
+{
```

Comment:
@bartnijssen and I discussed this.  We are planning to refactor `put_data` but as it is now, it doesn't meet the requirements we needed for coupling. This is an issue we'll want to come back to after the overhaul of `put_data()`